controllers/configmap_controller.go

```diff
@@ -100,8 +101,16 @@ func NewConfigMapReconciler(mgr manager.Manager, clusterConfig cluster.Config, w
 	if err != nil {
 		return nil, err
 	}
+	oc, err := openshiftClient.NewForConfig(mgr.GetConfig())
+	if err != nil {
+		return nil, errors.Wrap(err, "unable to create openshift client-go client")
+	}
+	ccmEnabled, err := cluster.IsCloudControllerOwnedByCCM(oc)
+	if err != nil {
+		return nil, errors.Wrap(err, "error determining if CCM owns cloud controller")
+	}
 	svcData, err := services.GenerateManifest(argsFromIgnition, clusterConfig.Network().VXLANPort(),
-		clusterConfig.Platform(), ctrl.Log.V(1).Enabled())
+		clusterConfig.Platform(), ccmEnabled, ctrl.Log.V(1).Enabled())
```

Comment:
Nit: I think I had mentioned this before but this parameter list is growing and it all has to do with cluster configuration, so a struct encapsulating that will be more succinct. But I am not blocking on that change.